core/src/test/java/io/questdb/test/griffin/wal/WalWriterFuzzTest.java

```diff
@@ -158,11 +158,30 @@ public void testWalWriteFullRandom() throws Exception {
     @Test
     public void testWalWriteFullRandomMultipleTables() throws Exception {
         Rnd rnd = TestUtils.generateRandom(LOG);
-        int tableCount = Math.max(2, rnd.nextInt(3));
+        int tableCount = Math.max(2, rnd.nextInt(4));
```

Comment:
The previous rnd boundary didn't make sense as it always led to `tableCount == 3`, so I changed it.